docs/en/observability/troubleshoot-uptime-mapping-issues.asciidoc

```diff
@@ -7,30 +7,31 @@
 == Mapping issues
 
 There are situations in which {heartbeat} data can be indexed without the correct mappings applied.
-These situations cannot occur with the {elastic-agent} configured via {fleet}, only with standalone {heartbeat}.
+These situations cannot occur with the {elastic-agent} configured via {fleet}, only with standalone {heartbeat} and {elastic-agent} running in standalone mode.
 This can occur when the underlying `heartbeat-VERSION` ILM alias is deleted manually or when {heartbeat} writes data
-through an intermediary such as {ls} without the `setup` command being run.
+through an intermediary such as {ls} without the `setup` command being run. 
+When running {elastic-agent} in standalone mode this can happen if manually setup datastreams have incorrect mappings.
```

Comment:
```suggestion
When running {elastic-agent} in standalone mode this can happen if manually-setup datastreams have incorrect mappings.
```

Reply:
I don't think this is grammatically more correct, do you have a strong feeling here? Maybe @bmorelli25 or @dedemorton have a thought